project_8.html

```diff
@@ -0,0 +1,793 @@
+					capture - converting
+					your prototyped design and circuit drawings into a digital schematic. (2) PCB Layout Design -
+					placing components and
+					routing conductive tracks in a virtual layout of your PCB. You can learn exactly how this process
+					works in the KiCad EDA
+					software by viewing our PCB Design with KiCad workshop material the lecture material <a
+						class="link-primary" href="./workshops.html" target="_blank" rel="noreferrer">here</a>.
+				</p>
+
+				<h2 class="mb-3 fw-bold" id="pj-heading-installing-kicad">Installing KiCAD 7.0</h2>
+
+				<p class="mb-3">
+					To complete this project, you will download the PCB design software KiCad (version 7.0.11).
+				</p>
+
+				<table class="table pj-download-table mb-5">
+					<tbody>
+						<tr>
+							<td>Windows</td>
+							<td>
+								<a class="btn btn-outline-success mx-3"
+									href="https://downloads.kicad.org/kicad/windows/explore/stable/download/kicad-7.0.11-x86_64.exe"
+									role="button"><i class="bi bi-download"></i> Download</a>
+							</td>
+						</tr>
+						<tr>
+							<td>Mac</td>
+							<td>
+								<a class="btn btn-outline-success mx-3"
+									href="https://downloads.kicad.org/kicad/macos/explore/stable/download/kicad-unified-universal-7.0.11.dmg"
+									role="button"><i class="bi bi-download"></i> Download</a>
+							</td>
+						</tr>
+					</tbody>
+				</table>
+
+				<ol class="list-group list-group-numbered mb-5">
+					<li class="list-group-item">
+						Download the installer from the link appropriate to your OS.
+					</li>
+					<li class="list-group-item">
+						Launch the installer. Click Next.
+						<div>
+							<img class="img-fluid my-4"
+								src=".\assets\images\project_8/ops_project_8_kicad_installer1.png">
+						</div>
+					</li>
+					<li class="list-group-item">
+						When the <strong>Choose Components</strong> dialog appears, make sure all but the “Help files”
+						components are selected. Then, click <strong>Next</strong>.
+						<div>
+							<img class="img-fluid my-4"
+								src=".\assets\images\project_8/ops_project_8_kicad_installer2.png">
+						</div>
+					</li>
+					<li class="list-group-item">
+						Select the <strong>Install Location</strong>. The default setting is usually acceptable. Then,
+						click <strong>Install</strong>.
+						<div>
+							<img class="img-fluid my-4"
+								src=".\assets\images\project_8/ops_project_8_kicad_installer3.png">
+						</div>
+					</li>
+					<li class="list-group-item">
+						The installation is complete! The fun starts now.
+					</li>
+				</ol>
+
+				<h2 class="mb-3 fw-bold" id="pj-heading-parts">Parts</h2>
+				<p class="mb-3">
+					This is not a physical parts list. The followig parts will be used in the PCB EDA application.
+				</p>
+
+				<!-- ======= Part Table ======= -->
+				<table class="table pj-part-table mb-5">
```

Comment:
Someone's table isn't tabling. The right outline/border is missing. I would recommend a little debugging. If all else fails, remove "pj-part-table" from the class attribute.

Reply:
the issue is because the footprint description is too long. I'll try to add some kind of text-wrapping

Reply:
fix rn: just reduce font size. However, table is broken for mobile.

Reply:
fixed. Added table responsive class: https://getbootstrap.com/docs/4.0/content/tables/#always-responsive